xdg-terminal-exec

```diff
@@ -92,9 +92,10 @@ find_any_entry(){
 		if [ -d "${DATA_DIR}/${DATA_PREFIX_DIR}" ]
 		then
 			debug "searching in \"${DATA_DIR}/${DATA_PREFIX_DIR}\""
-			find "${DATA_DIR}/${DATA_PREFIX_DIR}" -type f -iname "*.desktop" | while read ENTRY_FILE
```

Comment:
I could find a way to indicate from while subshell that outside loop (over DATA_DIR) should be stopped when we found valid entry.
So I made find result into variable and make while to run not in the subshell. That may another bad consequence though if find result is too long.